imports/plugins/included/sms/server/methods/sms.js

```diff
@@ -0,0 +1,86 @@
+import Twilio from "twilio";
+import Nexmo from "nexmo";
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Sms, Accounts } from "/lib/collections";
+import { Reaction, Logger } from "/server/api";
+
+
+/**
+ * Sms Methods
+ */
+Meteor.methods({
+  /**
+   * sms/saveSettings
+   * @summary This save the sms provider settings
+   * @param {Object} settings - settings
+   * @return {object} returns result
+   */
+  "sms/saveSettings": (settings) => {
+    check(settings, Object);
+    settings.shopId = Reaction.getShopId();
+    Meteor.subscribe("SmsSettings").ready();
```

Comment:
I don't think you want to call `ready` there. https://docs.meteor.com/api/pubsub.html#Subscription-ready